src/js/actions/userActions.js

```diff
@@ -595,3 +595,23 @@ export const setHideAnnouncement = (shouldHide, userId) => (dispatch, getState)
   }
   return Promise.resolve();
 };
+
+export const getTokens = () => (dispatch, getState) =>
+  GeneralApi.get(`${useradmApiUrl}/settings/tokens`).then(({ data: tokens }) => {
+    const user = getCurrentUser(getState());
+    const updatedUser = {
+      ...user,
+      tokens
+    };
+    return Promise.resolve(dispatch({ type: UserConstants.UPDATED_USER, user: updatedUser, userId: user.id }));
+  });
+
+const ONE_YEAR = 31536000;
+
+export const generateToken = name => dispatch =>
+  GeneralApi.post(`${useradmApiUrl}/settings/tokens`, { name, expires_in: ONE_YEAR })
+    .then(({ data: token }) => Promise.all([dispatch(getTokens()), token]))
+    .catch(err => commonErrorHandler(err, `There was an error creatin g the token:`, dispatch));
```

Comment:
Small typo here - remove space in "creatin g"

Reply:
That should now be fixed 👌